server/index.js

```diff
@@ -40,33 +40,21 @@ try {
 } catch (err) {}
 
 function onRequest (req, res) {
-  if (req.headers.host === 'tracker.webtorrent.io' ||
-      req.headers.host === 'tracker.webtorrent.io:' + config.ports.router.https) {
-    proxy.web(req, res, { target: 'http://127.0.0.1:' + config.ports.tracker })
-  } else if (req.headers.host === 'whiteboard.webtorrent.io' ||
-      req.headers.host === 'whiteboard.webtorrent.io:' + config.ports.router.https) {
-    proxy.web(req, res, { target: 'http://127.0.0.1:' + config.ports.whiteboard })
-  } else if (req.headers.host === 'peerdb.io') {
+  if (req.headers.host === 'peerdb.io') {
```

Comment:
Random side project I started working on. A key-value blob store that's zero-knowledge (encrypted), and uses WebTorrent for cheap P2P delivery. Unfinished. More info here: https://github.com/feross/peerdb

It's in it's own repo, but I was using the webtorrent.io IP address and sharing the same server. This should probably be removed.